core/src/main/resources/defaults/blueprints/7.2.7/cdp-sdx-medium-ha.bp

```diff
@@ -361,39 +377,39 @@
           },
           {
             "name": "default.replication.factor",
-            "value": "2"
+            "value": "3"
           },
           {
             "name": "offsets.topic.replication.factor",
-            "value": "2"
+            "value": "3"
           },
           {
             "name": "transaction.state.log.replication.factor",
-            "value": "2"
+            "value": "3"
           },
           {
             "name": "offset.storage.replication.factor",
-            "value": "2"
+            "value": "3"
           },
           {
             "name": "config.storage.replication.factor",
-            "value": "2"
+            "value": "3"
           },
           {
             "name": "status.storage.replication.factor",
-            "value": "2"
+            "value": "3"
           },
           {
             "name": "transaction.state.log.min.isr",
-            "value": "1"
+            "value": "2"
           },
           {
             "name": "min.insync.replicas",
-            "value": "1"
+            "value": "2"
           },
           {
             "name": "unclean.leader.election.enable",
```

Comment:
unclean.leader.election.enable defaults to false - you might not need to specify this parameter explicitly.

You can also consider omitting the config override for all the configs that now are set to the default values. Explicitly setting these is also perfectly fine.

Reply:
Ok.

Reply:
I removed them but default.replication.factor and min.insync.replicas weren't being set to the right value, so i've set it to default.replication.factor=3 and min.insync.replicas=2.

Reply:
LGTM now.